apps/style/code-studio/curriculum_catalog_container.module.scss

```diff
@@ -1,5 +1,19 @@
 @use "color.scss";
 
+.catalogFiltersContainer {
+  display: flex;
+  max-width: 60em;
+  margin: 1em auto;
+}
+
+.catalogFilterDropdown {
+  float: left;
+}
+
+.catalogClearFiltersButton {
+  margin-left: auto;
```

Comment:
Could you modify these styles to be direction agnostic? This is what I'm seeing for `rtl`

<img width="1082" alt="image" src="https://user-images.githubusercontent.com/9142121/234297741-737e526b-e5d9-4f9c-a10b-35d3be1eea2b.png">

Something like `display: flex` with `justify-content: space-between` might work if the "catalogFiltersContainer" has exactly two parts –– the filterDropdowns and the clearFiltersButton

Reply:
Or are you leaving the styling to Denys? Not sure what the style plan is

Reply:
Styling is up to Denys, this was just a hasty placeholder (which has since been removed)!